### ISSUE TITLE
Compatibility error between GoogleMaps and GoogleAnalytics plugins

### ISSUE THREAD

**Issue by bukosabino** [danwilson/google-analytics-plugin#512]
Hi all,

It looks like cordova-plugin-google-analytics and cordova-plugin-googlemaps aren't compatible with android platform.

Steps to reproduce:

```sh
ionic start nameApp blank
cd nameApp
ionic cordova platform add android
ionic cordova plugin add cordova-plugin-google-analytics
ionic cordova plugin add cordova-plugin-googlemaps --variable API_KEY_FOR_ANDROID="your_android_key" --variable API_KEY_FOR_IOS="your_ios_key"
ionic cordova build android
```

Stacktrace:

```
BUILD FAILED in 11s
27 actionable tasks: 6 executed, 21 up-to-date
(node:26949) UnhandledPromiseRejectionWarning: Error: /Users/your_path/nameApp/platforms/android/gradlew: Command failed with exit code 1 Error output:
/Users/your_path/nameApp/platforms/android/app/src/main/java/plugin/google/maps/PluginLocationService.java:497: error: cannot access zzbgl
    LocationRequest locationRequest= LocationRequest.create()
                                                    ^
  class file for com.google.android.gms.internal.zzbgl not found
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 11s
    at ChildProcess.whenDone (/Users/your_path/nameApp/platforms/android/cordova/node_modules/cordova-common/src/superspawn.js:169:23)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Socket.stream.socket.on (internal/child_process.js:346:11)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at Pipe._handle.close [as _onclose] (net.js:567:12)
(node:26949) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:26949) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by PandaBearz** [comment]
I am also having this issue, has anyone figured out why this is happening

**Comment by victorsosa** [comment]
please check issue #464 and fix in release 1.8.4
